react/FetchPrevious.js

```diff
@@ -22,7 +22,7 @@ const FetchPrevious = ({ htmlElementForButton = 'button' }) => {
     query: path(['variables', 'query'], searchQuery),
     map: path(['variables', 'map'], searchQuery),
     orderBy: path(['variables', 'orderBy'], searchQuery),
-    priceRange: path(['variables', 'priceRange'], searchQuery),
+    priceRange: path(['variables', 'selectedFacets'], searchQuery)?.find(facet => facet.key === 'priceRange')?.value,
```

Comment:
There's something weird, that is getting this change from your PR that already is on master.

https://github.com/vtex-apps/search-result/blob/master/react/FetchPrevious.js#L25

Reply:
@claudiu-iviteb